src/defaults/index.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
Since eslint isn't quite fully compatible with typescript, it doesn't understand how imported types are used, so it will throw errors here even if you've used the types in your code. What I've done here is admittedly a kinda messy workaround, but not sure what else I should do when I want to keep eslint and the `no-unused-vars` rule. 

I'm considering trying to switch from eslint to tslint though.